openedx/features/badging/constants.py

```diff
@@ -20,3 +20,7 @@
 INVALID_TEAM_ERROR = 'Cannot assign badge {badge_id} for invalid team {community_id}'
 TEAM_BADGE_ERROR = 'Cannot assign missing badges to user {user_id} in team {team_id}'
 UNKNOWN_COURSE_ERROR = 'Cannot assign badge {badge_id} for team {community_id} in unknown course'
+
+# philu badge notification type
+NOTIFICATION_TYPE = "{prefix}.{type}"
+NOTIFICATION_RENDERER = "edx_notifications.renderers.basic.JsonRenderer"
```

Comment:
Let's consider adding the `BADGE_` prefix in these names.

Reply:
well basically notification types could be added for other apps in future thats why I kept the name generic.

Reply:
This constant is in badging app, thats why `BADGE_` prefix seems redundant

Reply:
Being in the badging app is exactly the reason why I think it should have a `BADGE_` prefix, since we know it's strictly related to badging. Would greatly help when finding this specific variable and way lesser chances of there being a name conflict, because of the prefix that I suggested.

@monofal we should not be adding other app constants in here, because this is the constants file for the badging app.

Reply:
constant name changed to `EARNED_BADGE_NOTIFICATION_TYPE`